std/uni.d

```diff
@@ -713,7 +713,10 @@ import std.range.primitives; // back, ElementEncodingType, ElementType, empty,
     // save
 import std.traits; // isConvertibleToString, isIntegral, isSomeChar,
     // isSomeString, Unqual
-
+import std.exception;// : enforce;
+import core.memory; //: pureMalloc, pureRealloc, pureFree;
+import core.exception; // : onOutOfMemoryError;
+static import std.ascii;
```

Comment:
> What error do you get if you make those imports function local? Also if you can't make them function local, you should make them non-selective as currently there's a compiler bug causing them to behave as they were public.

@ZombineDev It's the other way around: every new non-selective import we add, is public.
In other words, the following is now possible:

```d
import std.uni : assumeUnique, pureMalloc; // ...
```

https://github.com/dlang/phobos/pull/5584#issuecomment-313993454

Reply:
Sorry I remembered that bug report backwards.